### PR TITLE
Update master pipeline

### PR DESCRIPTION
**Pull request by aarongilbert123**

### CODE REVIEW

File: Source/Relativity.Test.Helpers/Relativity.Test.Helpers.NUnit.Integration/Mail/GmailMailHelperTests.cs [hunk 1]
```diff
@@ -22,8 +22,10 @@ public class GmailMailHelperTests
 		private const string EmailDomain = "smtp.gmail.com";
 
 		// The following below should be changed for this test
-		private const string EmailAddress = "<YOUR_GMAIL_ADDRESS>";
-		private const string EmailPassword = "<YOUR_GMAIL_PASSWORD>";
+		//private const string EmailAddress = "<YOUR_GMAIL_ADDRESS>";
+		//private const string EmailPassword = "<YOUR_GMAIL_PASSWORD>";
```
Comment (TommyBigNose) [inline Source/Relativity.Test.Helpers/Relativity.Test.Helpers.NUnit.Integration/Mail/GmailMailHelperTests.cs]: Easy - Clean up extra commented lines

File: Source/Relativity.Test.Helpers/Relativity.Test.Helpers.NUnit.Integration/Mail/MailTrapMailHelperTests.cs [hunk 2]
```diff
@@ -25,10 +25,14 @@ public class MailTrapMailHelperTests
...
 
 		// The following below should be changed for this test
-		private const string ApiKey = "<YOUR_API_KEY>";
-		private const string EmailAddress = "<YOUR_TARGET_EMAIL>"; // Can really be any address since MailTrap will capture it
-		private const string EmailUsername = "<YOUR_USERNAME>";
-		private const string EmailPassword = "<YOUR_PASSWORD>";
+		//private const string ApiKey = "<YOUR_API_KEY>";
+		//private const string EmailAddress = "<YOUR_TARGET_EMAIL>"; // Can really be any address since MailTrap will capture it
+		//private const string EmailUsername = "<YOUR_USERNAME>";
+		//private const string EmailPassword = "<YOUR_PASSWORD>";
```
Comment (TommyBigNose) [inline Source/Relativity.Test.Helpers/Relativity.Test.Helpers.NUnit.Integration/Mail/MailTrapMailHelperTests.cs]: Easy - Clean up extra commented lines

File: Source/Relativity.Test.Helpers/Relativity.Test.Helpers.NUnit.Integration/Mail/GmailMailHelperTests.cs (hunk 1 above)
Comment (aarongilbert123) [reply Source/Relativity.Test.Helpers/Relativity.Test.Helpers.NUnit.Integration/Mail/GmailMailHelperTests.cs]: Done

File: Source/Relativity.Test.Helpers/Relativity.Test.Helpers.NUnit.Integration/Mail/MailTrapMailHelperTests.cs (hunk 2 above)
Comment (aarongilbert123) [reply Source/Relativity.Test.Helpers/Relativity.Test.Helpers.NUnit.Integration/Mail/MailTrapMailHelperTests.cs]: Done

**Pull request merged**